lib/models.ts

```diff
@@ -1,10 +1,17 @@
 import { Isotope, IsotopeSelect, IsotopeResult } from "isotopes";
 
 const createModel = <T extends Model>(name: string) =>
-  new Isotope<T>({
-    domain: `artsy-studio-${name}`,
-    key: "id"
-  });
+  new Isotope<T>(
+    {
+      domain: `artsy-studio-${name}`,
+      key: "id"
+    },
+    {
+      accessKeyId: process.env.ACCESS_KEY_ID,
+      secretAccessKey: process.env.SECRET_ACCESS_KEY,
+      region: process.env.REGION
+    }
+  );
```

Comment:
Why this change is required is a little weird. 

Now [restricts certain environment variables](https://zeit.co/docs/v2/platform/limits#reserved-variables) from being included in a build (like say... the AWS access token). [Isotopes](https://github.com/squidfunk/isotopes) is just a simple wrapper around [SimpleDB](https://aws.amazon.com/simpledb/) which we're using for a little metadata store. 

Isotopes doesn't actually support passing these yet, but I've got a [PR](https://github.com/squidfunk/isotopes/pull/466) open to add it. I patched it in w/ [patch-package](https://www.npmjs.com/package/patch-package) for now.